packages/showcases/css/stories/components/breadcrumb/examples/overview.html

```diff
@@ -0,0 +1,50 @@
+<div class="block">
+  <nav aria-label="Breadcrumb" class="vtmn-breadcrumb">
+    <ol>
+      <li><a href="">Home</a></li>
+      <li><a href="">Page 2</a></li>
+      <li><a href="" aria-current="page">Current</a></li>
```

Comment:
I think it will be better in this case to use "location" instead of "page".
See: https://www.aditus.io/patterns/breadcrumbs/ & https://www.aditus.io/aria/aria-current/

Reply:
Hmm, I prefer the documentation from W3C : https://www.w3.org/TR/wai-aria-practices/examples/breadcrumb/index.html